web/src/client/users.test.js

```diff
@@ -21,71 +21,91 @@
 
 // @ts-check
 
-import DBusClient from "./dbus";
+import { HTTPClient } from "./http";
 import { UsersClient } from "./users";
 
-jest.mock("./dbus");
+const mockJsonFn = jest.fn();
 
-const USERS_IFACE = "org.opensuse.Agama.Users1";
+const mockGetFn = jest.fn().mockImplementation(() => {
+  return {
+    ok: true,
+    json: mockJsonFn,
+  };
+});
 
-let setFirstUserResult = [true, []];
+const mockPutFn = jest.fn().mockImplementation(() => {
+  return {
+    ok: true,
+  };
+});
 
-const usersProxy = {
-  wait: jest.fn(),
-  FirstUser: ["Jane Doe", "jane", "12345", false, {}],
-  SetFirstUser: jest.fn().mockResolvedValue(setFirstUserResult),
-  RemoveFirstUser: jest.fn().mockResolvedValue(0),
-  SetRootPassword: jest.fn().mockResolvedValue(0),
-  RemoveRootPassword: jest.fn().mockResolvedValue(0),
-  RootPasswordSet: false,
-  SetRootSSHKey: jest.fn().mockResolvedValue(0),
-  RootSSHKey: "ssh-key"
-};
+const mockDeleteFn = jest.fn().mockImplementation(() => {
+  return {
+    ok: true,
+  };
+});
 
-beforeEach(() => {
-  // @ts-ignore
-  DBusClient.mockImplementation(() => {
-    return {
-      proxy: (iface) => {
-        if (iface === USERS_IFACE) return usersProxy;
-      }
-    };
-  });
+const mockPatchFn = jest.fn().mockImplementation(() => {
+  return {
+    ok: true,
+  };
+});
+
+jest.mock("./http", () => {
+  return {
+    HTTPClient: jest.fn().mockImplementation(() => {
+      return {
+        get: mockGetFn,
+        put: mockPutFn,
+        delete: mockDeleteFn,
+        patch: mockPatchFn,
+      };
+    }),
+  };
 });
 
+const FirstUser = {
```

Comment:
Please, use camelCase. For classes or React components it is right to use Pascal case, but not for regular variables and functions.